github-orgs/kubeflow/org.yaml

```diff
@@ -815,6 +815,14 @@ orgs:
             - varodrig
             - StefanoFioravanzo
             privacy: closed
+          kubeflow-notebooks-security:
```

Comment:
@ederign I think the team name that @andreyvelich created for was actually `notebooks-security`

Reply:
Fixed!